war/src/main/java/io/lumeer/engine/rest/dao/ViewDao.java

```diff
@@ -28,6 +33,9 @@
  */
 public class ViewDao {
```

Comment:
When doing refactoring, you can also rename this class to `ViewDto` or `ViewMetadata` which better describe what is it used for.